tests/client_test.py

```diff
@@ -472,3 +474,60 @@ def test_init_with_logging_config(self):
             Client.GoogleAdsClient(mock_credentials_instance,
                                    self.developer_token, logging_config=config)
             mock_dictConfig.assert_called_once_with(config)
+
+    def test_service_client_dot_enums_property(self):
```

Comment:
This test is a little hard to follow, but basically it tests the `service.enum` property for a service in every version of the API, which implicitly tests the `services/enums.py` file, which exposes the enums used by the property. It loops over each member of `service.enum` and compares it to the raw enum pb2 file found by calling `client.get_type` with the enum's name.

Reply:
Again, this comment belongs in the code rather than in the PR.

Reply:
Ok will add.